postgres_to_redshift.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"strings"
+
+	"github.com/clever/redshifter/postgres"
+	"github.com/clever/redshifter/redshift"
+	"github.com/segmentio/go-env"
+)
+
+var (
+	// TODO: include flag validation
+	awsRegion  = env.MustGet("AWS_REGION")
+	s3prefix   = flag.String("s3prefix", "", "s3 path to be used as a prefix for temporary storage of postgres data")
+	tables_csv = flag.String("tables", "", "Tables to copy as CSV")
+	dumppg     = flag.Bool("dumppostgres", true, "Whether to dump postgres")
+	updateRS   = flag.Bool("updateredshift", true, "Whether to replace redshift")
```

Comment:
Thoughts on env vs flags? It seems inconsistent. AWS key, secret, and region are from ENV. All else is from flags.

I think the long run discussion is here for getting to consistent env/flag patterns: https://clever.atlassian.net/browse/INFRA-503

One reason I mention it, too, is that I'm thinking of `postgres/postgres.go` as a library, so to me the NewDB constructor would make more sense to take arguments instead of read from flags. It's very convenient here, but is specific to being used in as a CLI app.